DailyCodingProblem/Problems1Through25.cs

```diff
@@ -125,19 +125,42 @@ public static int Problem4(int[] numbers)
         }
 
         /// <summary>
-        /// Returns the first element in a Pair
+        /// Returns the first element in a Pair.
         /// </summary>
         public static int Problem5GetFirst(Pair pair)
         {
             return pair.A;
         }
 
         /// <summary>
-        /// Returns the last element in a Pair
+        /// Returns the last element in a Pair.
         /// </summary>
         public static int Problem5GetLast(Pair pair)
         {
             return pair.B;
         }
+
+        /// <summary>
+        /// Converts byte RGB values to their Hexadecimal equivalent. 
+        /// Any RGB values that are outside of the byte range will be 
+        /// rounded to the closest valid byte value.
+        /// 
+        /// For example, the input (-5, 148, 300) would return "0094FF".
+        /// </summary>
+        public static string Problem6(int r, int g, int b)
+        {
+            return $"{ToByteRange(r):X2}{ToByteRange(g):X2}{ToByteRange(b):X2}";
+        }
+
+        private static int ToByteRange(int x)

```

Comment:
Why the name "ToByteRange"?

Reply:
So a byte is 8 bits, which can represent numbers from 0-255.

Here we are insuring the int that was passed in, is a valid byte value.

Reply:
Ok, I guess that is fine.